package.json

```diff
@@ -109,5 +109,13 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "mocha": {
```

Comment:
How about moving this to a `.mocharc.json` to keep the package.json smaller?
See: https://mochajs.org/#configuring-mocha-nodejs

Reply:
I will PR that :fist_raised:

Reply:
See: https://github.com/balena-io/open-balena-api/pull/374